app.rb

```diff
@@ -0,0 +1,46 @@
+require_relative './author'
+require_relative './game'
+require_relative './Modules/userInput'
```

Comment:
![requestchanges](https://github.com/torobucci/catalog-of-my-things/assets/26763433/2497770e-0ec5-4576-ad7e-2afc100cc6ea)

Reply:
@ReemMohamedAbdelfatah  Thanks for spotting the error. I have fixed the error